product_list_pricelist_price/models/product_template.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2021 ForgeFlow, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+
+from odoo import api, fields, models
+
+from odoo.addons.base.models.ir_ui_view import (
+    transfer_modifiers_to_node,
+    transfer_node_to_modifiers,
+)
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    def _compute_product_template_pricelist_price(self):
+        pricelists = self.env["product.pricelist"].search([("active", "=", True)])
+        for product in self:
+            result = pricelists._compute_price_rule_multi([(product, 1.0, False)])[
+                product.id
+            ]
+            for pricelist in pricelists:
+                field_name = "product_tmpl_price_pricelist_%s" % (pricelist.id)
+                product[field_name] = pricelist.currency_id.round(
+                    result[pricelist.id][0]
```

Comment:
@JordiMForgeFlow perhaps you should consider the case where pricelist.id is not in result, or that result[pricelist.id] does not contain an element? Just to avoid exceptions raised

Reply:
Based on the standard code it should always contain the result, as by default it will at least set a price of 0.0 for each product (https://github.com/odoo/odoo/blob/13.0/addons/product/models/product_pricelist.py#L170) and for each pricelist passed to the method _compute_price_rule_multi (https://github.com/odoo/odoo/blob/13.0/addons/product/models/product_pricelist.py#L91).

Nevertheless it doesn't hurt to add extra security, so I can add it 😄

Reply:
Done!